bundle/Resources/config/parameters.yaml

```diff
@@ -61,3 +61,6 @@ parameters:
     ngsite.default.template.menu: '@@KnpMenu/menu.html.twig'
 
     ngsite.phpstorm_twig_integration.enabled: '%kernel.debug%'
+
+    ngsite.default.download.content_types:
```

Comment:
This should be named `ngsite.default.direct_download.content_types`, because we're implementing a direct download feature here.

Reply:
fixed in ab4cac4df397dd851f944bfbe81cea0c743cf048